### DISCUSSION TITLE
Does Wix Toolset supports file Hashing while creating installer? If yes how to do that? Please help, thanks in advance.

### DISCUSSION THREAD

**Discussion by Agiliad** [wixtoolset/issues#6933] [Questions]
## Triage

Issues are triaged at [online meetings](https://www.firegiant.com/blog/), generally held alternate Thursdays at 9:30 a.m. Pacific time (UTC-7/UTC-8). Meeting requests are sent to the [wix-devs](http://wixtoolset.org/documentation/mailinglist/#wix-devs) and wix-users mailing lists, as well as in Discussions.

## Discussions and support

Note that discussions are best held on the mailing lists not in issues. Please do not open issues requesting support or debugging help; first start a thread on the [wix-users](http://wixtoolset.org/documentation/mailinglist/#wix-users) mailing list and open an issue if the discussion suggests a bug in the WiX Toolset or that a feature request is appropriate. You can also try out the beta [Discussions](https://github.com/wixtoolset/issues/discussions?discussions_q=category%3AQuestions) feature.

## Instructions

Read the sections above. Delete the Triage, Discussions and support, and Instructions sections and fill out the rest.

## Feature requests

If this issue is a feature request:

* Describe the scenario and benefits that the feature supports.

> (Provide lots of specific detail so we understand the context around the feature.)

* Describe how you're accomplishing the feature today (if possible).

> (Be as specific as you can and err on the side of providing too much information.)

* Describe what you'd like the new feature to do.

> (There's no need to design it but suggestions are appreciated.)

**Comment by barnson** [comment]
WiX doesn't use issues for support. We moved this issue to the [experimental GitHub discussions area][gh]. You might also consider asking your question on the [wix-users mailing list][ml] or posting on [Stack Overflow][so].

[gh]: https://github.com/wixtoolset/issues/discussions?discussions_q=category%3AQuestions
[ml]: http://wixtoolset.org/documentation/mailinglist/
[so]: https://stackoverflow.com/questions/tagged/wix

**Comment by robmen** [comment]
Yes. WiX hashes unversioned files using the `MsiFileHash()` function. The WiX Toolset is open source so you can see all of the code.

**Comment by Agiliad** [comment]
Thanks for the early response.
